packages/@aws-cdk/aws-scheduler-alpha/README.md

```diff
@@ -0,0 +1,369 @@
+<!--BEGIN STABILITY BANNER-->
+
+---
+
+![cdk-constructs: Experimental](https://img.shields.io/badge/cdk--constructs-experimental-important.svg?style=for-the-badge)
+
+> The APIs of higher level constructs in this module are experimental and under active development.
+> They are subject to non-backward compatible changes or removal in any future version. These are
+> not subject to the [Semantic Versioning](https://semver.org/) model and breaking changes will be
+> announced in the release notes. This means that while you may use them, you may need to update
+> your source code when upgrading to a newer version of this package.
+
+---
+
+<!--END STABILITY BANNER-->
+
+[Amazon EventBridge Scheduler](https://aws.amazon.com/blogs/compute/introducing-amazon-eventbridge-scheduler/) is a feature from Amazon EventBridge
+that allows you to create, run, and manage scheduled tasks at scale. With EventBridge Scheduler, you can schedule one-time or recurrently tens 
+of millions of tasks across many AWS services without provisioning or managing underlying infrastructure.
+
+1. **Schedule**: A schedule is the main resource you create, configure, and manage using Amazon EventBridge Scheduler. Every schedule has a schedule expression that determines when, and with what frequency, the schedule runs. EventBridge Scheduler supports three types of schedules: rate, cron, and one-time schedules. When you create a schedule, you configure a target for the schedule to invoke. 
+2. **Targets**: A target is an API operation that EventBridge Scheduler calls on your behalf every time your schedule runs. EventBridge Scheduler
+supports two types of targets: templated targets and universal targets. Templated targets invoke common API operations across a core groups of 
+services. For example, EventBridge Scheduler supports templated targets for invoking AWS Lambda Function or starting execution of Step Function state
+machine. For API operations that are not supported by templated targets you can use customizeable universal targets. Universal targets support calling
+more than 6,000 API operations across over 270 AWS services.
+3. **Schedule Group**: A schedule group is an Amazon EventBridge Scheduler resource that you use to organize your schedules. Your AWS account comes
+with a default scheduler group. A new schedule will always be added to a scheduling group. If you do not provide a scheduling group to add to, it 
+will be added to the default scheduling group. You can create up to 500 schedule groups in your AWS account. Groups can be used to organize the 
+schedules logically, access the schedule metrics and manage permissions at group granularity (see details below). Scheduling groups support tagging:
+with EventBridge Scheduler, you apply tags to schedule groups, not to individual schedules to organize your resources.
+
+This module is part of the [AWS Cloud Development Kit](https://github.com/aws/aws-cdk) project. It allows you to define Event Bridge Schedules.
+
```

Comment:
Lets add a big disclaimer up above everything letting people know this module is in active development and the contents of the readme are not implemented yet

Reply:
Removed parts that we haven't implemented yet and added a TODO with reference to RFC